BO4E/COM/StatusZusatzInformation.cs

```diff
@@ -0,0 +1,36 @@
+using System.Text.Json.Serialization;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Enhält das STS Segemt
```

Comment:
bitte hier eine menschenlesbare beschreibung, kein edifact-slang ;)

Reply:
(was heißt denn STS genau?)

Reply:
status? 😀 [In unser aller Lieblingsdokument](https://www.edi-energy.de/index.php?id=38&tx_bdew_bdew%5Buid%5D=1255&tx_bdew_bdew%5Baction%5D=download&tx_bdew_bdew%5Bcontroller%5D=Dokument&cHash=e7938f158eb9e0ab641e6524190893a9) auf Seite 41:
![grafik](https://user-images.githubusercontent.com/23094997/144852778-db70578a-a363-4d45-8344-2c81116ee028.png)

Reply:
aha. siehe update